### ISSUE TITLE
gtExtras required package

### ISSUE THREAD

**Issue by sje30** [grantmcdermott/quarto-revealjs-clean#11]
Thanks for this theme.

Just a small note: when I went to render the slides, I found as well as :

    install.packages(c("modelsummary", "gt", "fixest", "pdftools", "tinytex", "threejs"))

I also needed the gtExtras package (line 390)

Stephen

**Comment by grantmcdermott** [comment]
Thanks Stephen. I'm aware of this, but will be avoiding it altogether once the new version of `modelsummary` comes out since that switches to the new (and much lighter) `tinytable` backend. You can grab the dev versions of these packages and see the PR here if you want to kick the tyres for me. https://github.com/grantmcdermott/quarto-revealjs-clean-demo/pull/1

**Comment by sje30** [comment]
would be happy to test out.

Is it a question of 
```
devtools::install_github("vincentarelbundock/modelsummary")
devtools::install_github("vincentarelbundock/tinytable")
```

but how do I download your template?  Before I just used:

```
quarto install extension grantmcdermott/quarto-revealjs-clean
```

is there a way to specify that I should include the PR?

**Comment by grantmcdermott** [comment]
Sorry for dropping the ball on this @sje30. I was away on holiday.

Vincent has just released `modelsummary` 2.0.0 on CRAN  though and I've merged my PR. So you should be able to test directly without having to futz with dev versions.

(Please feel free to reopen if any other problems crop up.)
